recipe/setup.sh

```diff
@@ -22,6 +22,11 @@ if [ ! -d ./ZZMatrixElement ]; then
     popd
 fi
 
+if [ ! -d ./RecoMET ]; then
```

Comment:
How heavy and/or prone to break other people's code is this MET stuff? I'm currently working on making the HZZ stuff that tends to break everything optional; depending on how icky the MET recipes and code are we may want to do the same thing here. If we do, I'll make that change with my others.

Reply:
I wouldn't expect it to interfere with other people's stuff but it's possible. I would imagine most analyses would want to have the MET according to the current recipe, but I can make it optional.

Reply:
Does it slow the code down much? In the past, some MET stuff has.

Reply:
Not sure